api_reference.md

```diff
@@ -586,9 +607,9 @@ param = {
   }
 }
 
-q = Midtrans.update_subscription("d137e7f4-9474-4fc2-9847-672e09cb16f6", param)
+charge_response = Midtrans.update_subscription("d137e7f4-9474-4fc2-9847-672e09cb16f6", param)
 
-q == {
+charge_response.body == {
```

Comment:
why all these example is using `.body`? real case usage will be through `.data`, so maybe change the example format to

```
puts result.data # this will be Hash representation of the API JSON response, of example:
# <paste the json here>
```